components/style/properties/properties.mako.rs

```diff
@@ -1901,13 +1913,22 @@ pub mod style_structs {
                             self.${longhand.ident} = v;
                         }
                     % endif
-                    /// Set ${longhand.name} from other struct.
-                    #[allow(non_snake_case)]
-                    #[inline]
-                    pub fn copy_${longhand.ident}_from(&mut self, other: &Self) {
-                        self.${longhand.ident} = other.${longhand.ident}.clone();
-                    }
-
+                    % if longhand.ident == "display":
+                        /// Set ${longhand.name} from other struct.
```

Comment:
Ditto, let's make the comment a bit more specific, and document why the special case is needed.